src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -102,15 +102,16 @@ def wrapper(*args, **kwargs):
     # misuse partial to copy `wrapper`
```

Comment:
```suggestion
    # misuse lambda to copy `wrapper` function
```

Reply:
sure, will create a separate commit to address this.

Reply:
please don't worry too much. I simply missed adapting this code when trying out multiple different implementations and forgot to update this comment after being done. 

I simply though it would make sense to resolve this, as you're touching the code anyway.

Reply:
add the commit for the stale comment.